srcpkgs/cgo/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'cgo'
+pkgname=cgo
+version=0.5.0
+revision=1
+build_style=gnu-makefile
+short_desc="Terminal based gopher client"
+maintainer="Benjamín Albiñana <benalb@gmail.com>"
+license="ISC"
+homepage="https://github.com/kieselsteini/cgo"
+distfiles="https://github.com/kieselsteini/cgo/archive/v${version}.tar.gz"
+checksum=89e4038d27dae0e009c7d313fc1945c5ee52006685b517ec0970b198e098ad54
+
+if [ "$CROSS_BUILD" ]; then
+	pre_build() {
```

Comment:
I think you can enable the `pre_build()` for all targets, not just cross build; CC is always defined in the build environment.

Reply:
ok, will remove the if loop and push again. :)